man/systemd.netdev.xml

```diff
@@ -600,8 +600,8 @@
       <listitem>
         <para>Configures the default destination UDP port on a per-device basis.
         If destination port is not specified then Linux kernel default will be used.
-        Set destination port 4789 to get the IANA assigned value,
-        and destination port 0 to get default values.</para>
+        Set destination port 4789 to get the IANA assigned value. If destination
+        port is set to 0 then default value (4789) is used.</para>
```

Comment:
Nah, this is not correct: if you specify the port as "0", this should result in a parse failure now, and rightfully so. However, if you assign the empty string, then you get the IANA default value. These two sentences should hence be replaced by something like this:

```If not set or If the destination port is assigned the empty string the default port of 4789 is used.```